### ISSUE TITLE
fst --help Error

### ISSUE THREAD

**Issue by Ellie-BM** [veg/hyphy#1274]
I would really like to know all the command line inputs for `hyphy fst` so I don't have to use the interactive mode anymore. 

When I type in the command `hyphy fst --help` I get the following message

```
Available analysis command line options
---------------------------------------
Use --option VALUE syntax to invoke
If a [reqired] option is not provided on the command line, the analysis will prompt for its value
[conditionally required] options may or not be required based on the values of other options

Error:
A recursive dependency error in _Variable::Compute; this is an HBL implementation bug; offending variable is 'MESSAGE_LOGGING'

Check errors.log for execution error details.
```

In `errors.log`, there is the message

```
Error:A recursive dependency error in _Variable::Compute; this is an HBL implementation bug; offending variable is 'MESSAGE_LOGGING'
A recursive dependency error in _Variable::Compute; this is an HBL implementation bug; offending variable is 'MESSAGE_LOGGING'
```

Thanks for the help.

**Comment by spond** [comment]
Dear @Ellie-BM,

Good catch! `fst` is not something I have upgraded to use CLI arguments. But since you found this odd issue, which I will fix, I will also port `fst` to use command line arguments. 

Stay tuned.

Best,
Sergei

**Comment by Ellie-BM** [comment]
Great, looking forward to it, I had made a script to run through the interactive mode, but full command line options seem much better.

Ellie

**Comment by spond** [comment]
Dear @Ellie-BM,

These issues should have been addressed with release 2.5.27.

```
hyphy fst --help

hyphy % ./hyphy fst --help

Available analysis command line options
---------------------------------------
Use --option VALUE syntax to invoke
If a [reqired] option is not provided on the command line, the analysis will prompt for its value
[conditionally required] options may or not be required based on the values of other options

distances
	Distance Computation
	default value: Distance formulae

names
	Sequence names
	default value: Included

type
	Data type
	default value: Nucleotide/Protein

code
	Genetic Code
	default value: Universal

alignment [required]
	Sequence alignment

matrix [required]
	Load the names/distance matrix

regexp1 [required]
	Regular expression for the first sequence partition

regexp2 [required]
	Regular expression for the second sequence partition

formula
	Distance Formula
	default value: JC69

bootstrap
	Bootstrap Estimators
	default value: Sure

bootstrap-samples
	The number of bootstrap replicates to draw
	default value: 100 [computed at run time]

permutation
	Permutation Test
	default value: But of course

permutation-samples
	The number of permutations to perform
	default value: 100 [computed at run time]
```

Best,
Sergei
